<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2423</width>
    <height>1474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>2508</width>
         <height>1397</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="labelPNG">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>71</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>248</height>
               </size>
              </property>
              <property name="title">
               <string>Gateway config</string>
              </property>
              <widget class="QPushButton" name="connectbutton">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>100</y>
                 <width>131</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>131</width>
                 <height>21</height>
                </size>
               </property>
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>129</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>129</width>
                 <height>26</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="iplabel">
                  <property name="minimumSize">
                   <size>
                    <width>13</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>IP:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ipTextBox">
                  <property name="minimumSize">
                   <size>
                    <width>108</width>
                    <height>24</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>11</x>
                 <y>61</y>
                 <width>141</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>141</width>
                 <height>26</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="portlabel">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Port:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="portTextBox">
                  <property name="minimumSize">
                   <size>
                    <width>108</width>
                    <height>24</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>130</y>
                 <width>391</width>
                 <height>86</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QPushButton" name="startbutton">
                  <property name="minimumSize">
                   <size>
                    <width>389</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>START</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stopbutton">
                  <property name="minimumSize">
                   <size>
                    <width>389</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>STOP</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="SET">
                  <property name="minimumSize">
                   <size>
                    <width>389</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>SET</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>248</height>
               </size>
              </property>
              <property name="title">
               <string>Dataset</string>
              </property>
              <widget class="QPushButton" name="loadDirectory">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>40</y>
                 <width>171</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>171</width>
                 <height>21</height>
                </size>
               </property>
               <property name="text">
                <string>Load Directory</string>
               </property>
              </widget>
              <widget class="QComboBox" name="directoryBox">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>80</y>
                 <width>121</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>121</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
              <widget class="QComboBox" name="patientBox">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>100</y>
                 <width>121</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>121</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="sujeitoLabel">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>100</y>
                 <width>61</width>
                 <height>24</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>61</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Sujeito:</string>
               </property>
              </widget>
              <widget class="QLabel" name="atividadeLabel">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>80</y>
                 <width>71</width>
                 <height>24</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>71</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Atividade:</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>248</height>
               </size>
              </property>
              <property name="title">
               <string>LOGGER</string>
              </property>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>371</width>
                 <height>151</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>291</width>
                 <height>101</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>248</height>
               </size>
              </property>
              <property name="title">
               <string>Dados</string>
              </property>
              <widget class="QLabel" name="labelTipoAtividade">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>30</y>
                 <width>91</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>91</width>
                 <height>51</height>
                </size>
               </property>
               <property name="text">
                <string>Atividade Atual:</string>
               </property>
              </widget>
              <widget class="QLabel" name="labelAtividadeDado">
               <property name="geometry">
                <rect>
                 <x>120</x>
                 <y>30</y>
                 <width>301</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>301</width>
                 <height>51</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="lDado">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>200</y>
                 <width>261</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>241</width>
                 <height>51</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="labelLocalizacaoAtual">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>220</y>
                 <width>101</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>Localizacao Atual:</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="minimumSize">
               <size>
                <width>391</width>
                <height>248</height>
               </size>
              </property>
              <property name="title">
               <string>Sensores</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>60</y>
                 <width>71</width>
                 <height>80</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QCheckBox" name="AccBox">
                  <property name="text">
                   <string>AccBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="GyrBox">
                  <property name="text">
                   <string>GyrBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="MagBox">
                  <property name="text">
                   <string>MagBox</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_10">
              <property name="title">
               <string>GroupBox</string>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>60</y>
                 <width>181</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="text">
                <string>POLITECNICA PUCRS</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>210</x>
                 <y>30</y>
                 <width>141</width>
                 <height>161</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../../Users/Chines/Pictures/images.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox_33">
                <property name="minimumSize">
                 <size>
                  <width>1183</width>
                  <height>197</height>
                 </size>
                </property>
                <property name="title">
                 <string>acc_x</string>
                </property>
                <widget class="QCustomPlot" name="acc_x_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>19</y>
                   <width>1171</width>
                   <height>171</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>171</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_34">
                <property name="minimumSize">
                 <size>
                  <width>1183</width>
                  <height>196</height>
                 </size>
                </property>
                <property name="title">
                 <string>acc_y</string>
                </property>
                <widget class="QCustomPlot" name="acc_y_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>19</y>
                   <width>1161</width>
                   <height>171</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1161</width>
                   <height>171</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_35">
                <property name="minimumSize">
                 <size>
                  <width>1183</width>
                  <height>197</height>
                 </size>
                </property>
                <property name="title">
                 <string>acc_z</string>
                </property>
                <widget class="QCustomPlot" name="acc_z_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>30</y>
                   <width>1161</width>
                   <height>161</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1161</width>
                   <height>161</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_36">
                <property name="minimumSize">
                 <size>
                  <width>1183</width>
                  <height>196</height>
                 </size>
                </property>
                <property name="title">
                 <string>gyr_x</string>
                </property>
                <widget class="QCustomPlot" name="gyr_x_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>29</y>
                   <width>1161</width>
                   <height>161</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1161</width>
                   <height>161</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_37">
                <property name="minimumSize">
                 <size>
                  <width>1183</width>
                  <height>197</height>
                 </size>
                </property>
                <property name="title">
                 <string>gyr_y</string>
                </property>
                <widget class="QCustomPlot" name="gyr_y_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>2</x>
                   <y>21</y>
                   <width>1171</width>
                   <height>171</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>171</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="minimumSize">
                 <size>
                  <width>1186</width>
                  <height>247</height>
                 </size>
                </property>
                <property name="title">
                 <string>gyr_z</string>
                </property>
                <widget class="QCustomPlot" name="gyr_z_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>29</y>
                   <width>1171</width>
                   <height>211</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>211</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="minimumSize">
                 <size>
                  <width>1186</width>
                  <height>248</height>
                 </size>
                </property>
                <property name="title">
                 <string>azimuth</string>
                </property>
                <widget class="QCustomPlot" name="azimuth_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>29</y>
                   <width>1171</width>
                   <height>211</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>211</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="minimumSize">
                 <size>
                  <width>1186</width>
                  <height>248</height>
                 </size>
                </property>
                <property name="title">
                 <string>pitch</string>
                </property>
                <widget class="QCustomPlot" name="pitch_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>29</y>
                   <width>1171</width>
                   <height>221</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>221</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="minimumSize">
                 <size>
                  <width>1186</width>
                  <height>248</height>
                 </size>
                </property>
                <property name="title">
                 <string>roll</string>
                </property>
                <widget class="QCustomPlot" name="roll_widget" native="true">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>29</y>
                   <width>1171</width>
                   <height>211</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>1171</width>
                   <height>211</height>
                  </size>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
